server/server.go

```diff
@@ -1470,9 +1474,12 @@ func configureBackends(backends map[string]*types.Backend) {
 	}
 }
 
-func (s *Server) registerMetricClients(metricsConfig *types.Metrics) {
-	var registries []metrics.Registry
+func (*Server) registerMetricClients(metricsConfig *types.Metrics) metrics.Registry {
```

Comment:
As you don't use receiver, you can use only a func instead of a method

Reply:
Good point, fixed.